### ISSUE TITLE
typo in "interface" ?

### ISSUE THREAD

**Issue by slootsky** [zwack-am/pi-pm3#1]
in the source v0.1.0.tar.gz  file pi-pm3-0.1.0/pi-gen/stageDTPM3/02-Wireless-AP/00-run-chroot.sh

I think it has a typo in it on line 5

inteface wlan0

instead of interface  (the R)

**Comment by zwack-am** [comment]
Yes...  You are better off pulling down just one of the release images, or the git repo.

The newer repo copies are what I built from, I still need to work more on the 32 bit version.

Thanks,

Adam

On April 9, 2022 10:06:10 PM PDT, slootsky ***@***.***> wrote:
>in the source v0.1.0.tar.gz  file pi-pm3-0.1.0/pi-gen/stageDTPM3/02-Wireless-AP/00-run-chroot.sh
>
>I think it has a typo in it on line 5
>
>inteface wlan0
>
>instead of interface  (the R)
>
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/zwack-am/pi-pm3/issues/1
>You are receiving this because you are subscribed to this thread.
>
>Message ID: ***@***.***>
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
